### PR TITLE
Systelab spinner is disabled

### PR DESCRIPTION
**Pull request by olgapuig**
Add isDisabled property

### CODE REVIEW

**Comment by olgapuig** [comment]
Ready to be merged

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components",
-  "version": "5.3.7",
+  "version": "5.4.7",
```
Comment (alfonsserra) [inline package.json]: After the 5.3.7, the following version should be 5.4.0.

File: src/app/systelab-components/spinner/spinner.component.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { TouchSpinValues } from './touch.spin-values';
 })
 export class TouchspinComponent {
 
-	public isDisabled = false;
+	@Input() public isDisabled = false;
```
Comment (alfonsserra) [inline src/app/systelab-components/spinner/spinner.component.ts]: It should be consistent across components. There are 3 of them with "disabled" (switch, datepicker and toggle) and 4 with "isDisabled" (combo, searcher, month and week selector).

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ In order to clone the repository and test the library use the following commands
 git clone https://github.com/systelab/systelab-components.git
 cd systelab-components
 npm install
-ng serve
+ng serve (or npm run ng serve)
```
Comment (alfonsserra) [inline README.md]: install angular cli globally or run it form the root folder

File: package.json (hunk 1 above)
Comment (olgapuig) [reply package.json]: ok, it's done.

**Pull request merged**